### DISCUSSION TITLE
"User type can't be blank" from https://education.github.com/discount_requests/application

### DISCUSSION THREAD

**Discussion by tc-git-1** [community/community#63502] [GitHub Education]
### Hi everyone,

I went to https://education.github.com/discount_requests/application 

I completed the form
- I selected "Student"
- I used a ".edu" email address

I clicked the green "Continue" button

I then get this message:
"User type can't be blank"

Questions:
Q. How do I resolve this problem?
Q. How do I update my "user type"?

THANKS!

**Comment by safnimj** [comment]
I think you can try with another email of yours

**Reply by GhostOf0days** [reply]
I've seen consistent responses from you where you give non-correct or otherwise low quality answers. Please do not answer questions you do not know about. This is more so to keep the quality of discussions intact. I appreciate you taking the time to answer questions, but please put a little more research and effort into your answers. Thank you.
